src/runner.js

```diff
@@ -48,6 +48,8 @@ export default async function runPerformanceTest(
         logLevel: 'trace',
         outputDir: logDir,
         headers: { 'User-Agent': SPEEDO_WD_UA },
+        connectionRetryCount: 1,
```

Comment:
The default for this is `3` any reason we reduce this to `1`?

Reply:
Do we need 3 retries here? It looks like we already have retry logic in runner file. https://github.com/saucelabs/speedo/blob/master/src/runner.js#L8

Reply:
One is retrying a single command while the other retries almost the whole flow. According to the issue mentioned above the problem comes from the url command and with that the process of capturing the performance of the page load is not finishing in time. This seems because we don't see the CPU or network to idle to finish the capturing process which makes sense for large sites like BestBuy. The question is how likely is it that a retry will cause our system to successfully capture the performance. That depends on the side and threshold, in this case I doubt that it would have helped in others it might. That said increasing the `connectionRetryTimeout` will increase the overall time we come to a result and with 3 retries a user would wait 9 minutes for the command to ultimately fail. Reducing it to one retry therefor makes sense.